R/series__series.R

```diff
@@ -40,6 +45,34 @@
 #'   that we can use this information later on to avoid re-sorting it.
 #' - `SORTED_DESC` is similar but applies to sort in decreasing order.
 #'
+#' ## name
+#'
+#' `$name` returns the name of the Series.
+#'
+#' ## shape
+#'
+#' `$shape` returns a numeric vector of length two with the number of length
+#' of the Series and width of the Series (always 1).
+#'
+#' @section Sub-namespaces:
+#'
+#' Some functions are stored under active bindings, so it works like a sub-namespaces.
+#'
+#' ## expr
+#'
+#' `$expr` works as a workaround for applying arbitrary [Expr][Expr_class] methods
+#' to the Series class object, which means that this is a shortcut
+#' works like `pl$select(s)$select(pl$col(s$name)$<Expr method>)$to_series(0)`.
+#' This subnamespace is experimental.
+#'
```

Comment:
Sorry if I'm missing something obvious but I don't understand the point of this subnamespace. In the examples, you call `pl$Series(c(1:3))$expr$add(1)`, but that is simply the same as calling `pl$Series(c(1:3))$add(1)`. Why do we need this `$expr`?

Reply:
I don't think `$expr` was added by me. Why it was added I do not know.

Reply:
This PR is just a refactoring, and I don't think we should discuss its significance now.
We can delete this later.

Reply:
Ah ok, I didn't notice that was already there

Reply:
Yes, no one even noticed that $list methods were returning a DataFrame until now, so I'm guessing that very few users used such a method in Series